birdhouse/config/thredds/default.env

```diff
@@ -6,10 +6,28 @@ export THREDDS_DOCKER=pavics/thredds-docker
 export THREDDS_IMAGE="${THREDDS_DOCKER}:${THREDDS_VERSION}"
 export THREDDS_ORGANIZATION="Birdhouse"
 
+# All default values for data locations are set to ensure backward compatibility
```

Comment:
We have an unofficial convention that I like: start environment variables with the component name. That way it is really easy to determine which component each variable applies to. 

With that in mind, I'd recommend renaming some of these to:

```sh
export THREDDS_DATA_LOCATION_ON_CONTAINER='/pavics-ncml'
export THREDDS_RAW_DATA_LOCATION_ON_CONTAINER='/pavics-data'
export THREDDS_DATA_LOCATION_ON_HOST='${DATA_PERSIST_ROOT}/ncml'  # this default is for backward compatibility 
export THREDDS_RAW_DATA_LOCATION_ON_HOST='${DATA_PERSIST_ROOT}/datasets'  # this default is for backward compatibility
export THREDDS_DATA_LOCATION_NAME='Datasets'  # this default is for backward compatibility 
export THREDDS_RAW_DATA_LOCATION_NAME='Birdhouse'  # this default is for backward compatibility 
export THREDDS_DATA_LOCATION_NAME_lower=$(echo "$DATA_LOCATION_NAME" | tr '[:upper:]' '[:lower:]')
export THREDDS_RAW_DATA_LOCATION_NAME_lower=$(echo "$RAW_DATA_LOCATION_NAME" | tr '[:upper:]' '[:lower:]')
```